<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>1121</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="mainMenuPage">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="mainTitle">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>230</y>
        <width>471</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Candara</family>
        <pointsize>28</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(255, 255, 255, 0.8);
border-image: none;
font: 75 28pt &quot;Candara&quot;;</string>
      </property>
      <property name="text">
       <string>Minisweeper Agent</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="mainMenuGroup">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>300</y>
        <width>291</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: none;
border: none;
font: 20pt &quot;Courier New&quot;;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="exitBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>251</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(211, 215, 240);
color: #000000;
border: 2px solid #000;
border-radius: 30px;</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
      <widget class="QPushButton" name="playGameBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>251</width>
         <height>61</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>84</red>
             <green>187</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>84</red>
             <green>187</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>84</red>
             <green>187</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>84</red>
             <green>187</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>84</red>
             <green>187</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>84</red>
             <green>187</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>84</red>
             <green>187</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>84</red>
             <green>187</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>84</red>
             <green>187</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 30px;
background-color: rgb(84, 187, 255);
border: 2px solid #000;
</string>
       </property>
       <property name="text">
        <string>Play Game</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>120</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(&quot;resource/image/icon.png&quot;) 0 0 0 0 stretch stretch;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gamePage">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>581</width>
        <height>551</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="field"/>
     </widget>
     <widget class="QLabel" name="gameTitle">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Minesweeper</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="quitGameBtn">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>170</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Quit Game</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startGameBtn">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>170</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Game</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="tcPathField">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>690</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="loadTCBtn">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>690</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Load TC</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="historyField">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>250</y>
        <width>251</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="delayTextEdit">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>130</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="nextMoveBtn">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>210</y>
        <width>112</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Next Move</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StreamToggleBtn">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>210</y>
        <width>112</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Stream</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delayPlayBtn">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>130</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Delay</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
